Lib/multiprocessing/resource_tracker.py

```diff
@@ -212,6 +212,15 @@ def _send(self, cmd, name, rtype):
         assert nbytes == len(msg), "nbytes {0:n} but len(msg) {1:n}".format(
             nbytes, len(msg))
 
+    def __del__(self):
+        # making sure child processess are cleaned before ResourceTracker
+        # gets destructed.
+        # see https://github.com/python/cpython/issues/88887
+        try:
+            self._stop()
+        except:
```

Comment:
Catching some exceptions is fine. But catching any exception is not ok. You should not ignore KeyboardInterrupt for example.